src/spaceObjects/spaceship.cpp

```diff
@@ -56,10 +56,14 @@ REGISTER_SCRIPT_SUBCLASS_NO_CREATE(SpaceShip, ShipTemplateBasedObject)
     REGISTER_SCRIPT_CLASS_FUNCTION(SpaceShip, setSystemCoolant);
     REGISTER_SCRIPT_CLASS_FUNCTION(SpaceShip, getImpulseMaxSpeed);
     REGISTER_SCRIPT_CLASS_FUNCTION(SpaceShip, setImpulseMaxSpeed);
```

Comment:
I believe here you are not backwards compatible. If someone in his mission script changed speed and acceleration of some ship, it would now change only for forward speed and it will keep reverse speeds intact, would it not? 

Just want it to be clarified (so it can be specified in documentation, that there were change in the behavior).

Reply:
The main idea is : if only using setSpeed, then forward speed is always the same as reverse speed
If set reverseSpeed, then reverseSpeed is always inherited and not overwritten by a new "setSpeed".
Still, using an optional in setSpeed with two optional arguments would be better